app/controllers/users_controller.rb

```diff
@@ -0,0 +1,41 @@
+class UsersController < ApplicationController
+
+  def sign_up
+  end
+
+  def sign_up!
```

Comment:
one thing to note, is this code will severely conflict with the devise installation.